dist/openshift/cincinnati.yaml

```diff
@@ -229,9 +229,9 @@ parameters:
     displayName: cincinnati version
     description: cincinnati version which defaults to latest
   - name: GB_MEMORY_LIMIT
-    value: "256Mi"
+    value: "512Mi"
```

Comment:
I am interested to find out why we need this bump. Is this expected from https://github.com/openshift/cincinnati/pull/174

Reply:
If we change the currency level to 8 can it be done with in 256. @steveeJ ^

Reply:
in order to find out what's chewing RAM we need to stop OOMKills first

Reply:
https://github.com/openshift/cincinnati/pull/174#discussion_r375927706 so the new cache needs more memory.

Reply:
#174 doesn't increase the final cache size, it just fills it up concurrently which uses more memory. I'm curious if it fits into 512MiB as well, let's try it.